thundra/integrations/modules/requests.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+import wrapt
+from thundra import utils
+from thundra import constants
+from thundra.integrations.modules.generic_wrapper import wrapper
+from thundra.integrations.events.requests import RequestsEventFactory
+
+def _wrapper(wrapped, instance, args, kwargs):
+    prepared_request = args[0]
+
+    if utils.is_excluded_url(prepared_request.url):
+        return wrapped(*args, **kwargs)
+
+    return wrapper(RequestsEventFactory, wrapped, instance, args, kwargs)
+
+def patch():
+    disable_http_integration_by_env = utils.get_configuration(constants.THUNDRA_DISABLE_HTTP_INTEGRATION)
+    if not utils.should_disable(disable_http_integration_by_env):
+        wrapt.wrap_function_wrapper(
```

Comment:
Do we cover sessionless requests? I mean there might be calls over `requests` like `requests.post(...)`.

Reply:
Yes, i think that that function is called even if the request is sessionless. Also our tests also make sessionless requests and they are working.

Reply:
Ok, can you also add tests for sending HTTP requests over session?